endpoints_management/control/wsgi.py

```diff
@@ -395,6 +404,33 @@ def _create_check_info(self, method_info, parsed_uri, environ):
         )
         return check_info
 
+    def _create_quota_info(self, method_info, parsed_uri, environ):
+        service_name = environ.get(EnvironmentMiddleware.SERVICE_NAME)
+        operation_id = self._next_operation_id()
+        api_key_valid = False
+        api_key = _find_api_key_param(method_info, parsed_uri)
```

Comment:
Isn't there a helper for this API key logic?

Reply:
I don't think so, but I added one.